tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.AssemblyReferencesHaveNotChanged.net6.0.verified.txt

```diff
@@ -49,4 +49,5 @@ System.Text.RegularExpressions, Version=6.0.0.0, Culture=neutral, PublicKeyToken
 System.Threading, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
 System.Threading.Thread, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
 System.Threading.ThreadPool, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
+System.Web.HttpUtility, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
```

Comment:
Oh no, new assembly version 🙁 We definitely don't want a System.Web reference in .NET 6/.NET Core app 3.1.

Reply:
+1

Reply:
Fixed in 45b5532cc4b678569a1a7fca0c9483c9353dfb13